server/src/main/java/org/candlepin/resource/JobResource.java

```diff
@@ -174,6 +181,45 @@ public SchedulerStatus setSchedulerStatus(boolean running) {
         return getSchedulerStatus();
     }
 
+    @POST
+    @Path("schedule/{task}")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    @SuppressWarnings("unchecked")
+    public JobStatus schedule(@PathParam("task") String task) {
+
+        /*
+         * at the time of implementing this API, the only jobs that
+         * are permissible are cron jobs.
+         */
+        String className = "org.candlepin.pinsetter.tasks." + task;
+        try {
+            boolean allowed = false;
+            for (String permissibleJob : ConfigProperties.DEFAULT_TASK_LIST) {
+                if (className.equalsIgnoreCase(permissibleJob)) {
+                    allowed = true;
+                    // helps ignore case
+                    className = permissibleJob;
+                }
+            }
+            if (allowed) {
+                Class taskClass = Class.forName(className);
+                return pk.scheduleSingleJob((Class<? extends KingpinJob>) taskClass, Util.generateUUID());
+            }
+            else {
+                throw new BadRequestException(i18n.tr("Not a permissible job: {0}", task));
```

Comment:
In the next steps for this overarching task, we should revisit this error message and focus a bit more on the "why" it's not allowed to be triggered manually. Might require more work than just an else-branch though.